debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java

```diff
@@ -371,5 +372,21 @@ public void sleep(long ms) {
                 // Ignore
             }
         }
+
+        public void waitObject(Object obj, Supplier<Boolean> condition, long deadlineMs) throws InterruptedException {
```

Comment:
Where's this used?

Reply:
@gunnarmorling It is an impleemntation of the new method added by the interface

Reply:
Ok; so is there any reason for not simply using the existing `SystemTime` class? I cannot see any difference to our implementation.

Reply:
@gunnarmorling Removed, probably present for historical reasons